tests/native_tests/language.hy

```diff
@@ -552,9 +552,7 @@
   (assert (= (.a.b.meth x) "meth"))
   (assert (= (.a.b.meth x "foo" "bar") "meth foo bar"))
   (assert (= (.a.b.meth :b "1" :a "2" x "foo" "bar") "meth foo bar 2 1"))
-  (assert (= (.a.b.meth x #* ["foo" "bar"]) "meth foo bar"))
-
-  (assert (is (.isdigit :foo) False)))
```

Comment:
If you don't replace this line, then we're no longer testing whether `(.method :keyword)` is interpreted as intended.

Reply:
Good catch, also deleted too much...